### PR TITLE
Add deployments package

### PR DESCRIPTION
**Pull request by ronin13**
The package provides  DockerImageURLForService for obtaining a
docker image url when a service name and a deployment group have been
provided.

### CODE REVIEW

File: deployments/deployments.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+package deployments
+
+import (
+	"fmt"
+
+	paastaconfig "github.com/Yelp/paasta-tools-go/config"
+)
+
+const (
+	dockerRepo = "docker-paasta.yelpcorp.com:443"
```
Comment (Rob-Johnson) [inline deployments/deployments.go]: don't hardcode this. there is another system-paasta-config you can get it from. I'd parameterize `getImageURL` with a docker registry address

Comment (ronin13) [reply deployments/deployments.go]: Using the docker registry file now.

File: deployments/deployments.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+// DockerRegistry ...
+type DockerRegistry struct {
+	Registry string `json:"docker_registry"`
+}
+
+func getDockerRegistry() (string, error) {
+	dockerRegistry := &DockerRegistry{}
+	configreader := paastaconfig.SystemPaaSTAConfigFileReader{
```
Comment (Rob-Johnson) [inline deployments/deployments.go]: this doesn't look right to me - won't this end up trying to read `/etc/paasta/docker_registry.json/deployments.json`?

Comment (ronin13) [reply deployments/deployments.go]: My bad, seems like an oversight. Fixed. (I will also test this locally once)

**Pull request merged**